src/Persistence/Sql/Join.php

```diff
@@ -250,8 +250,10 @@ public function doDelete($id): void
             return;
         }
 
-        $id = $this->reverse ? $this->getOwner()->getId() : $this->getOwner()->get($this->master_field);
+        $model = $this->getOwner();
+        $query = $this->dsql();
```

Comment:
I dont see the use of putting these 2 in variables. Whats the idea behind this? Better code readability?

Reply:
Yeah, this is not functional change. Just made it a bit more similar to other methods above :)